hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -473,8 +473,8 @@ public void saveToCleanRequested(HoodieInstant instant, Option<byte[]> content)
     Preconditions.checkArgument(instant.getState().equals(State.REQUESTED));
     // Write workload to auxiliary folder
     createFileInAuxiliaryFolder(instant, content);
-    // Plan is only stored in auxiliary folder
-    createFileInMetaPath(instant.getFileName(), Option.empty(), false);
+    // Plan is stored in meta path
+    createFileInMetaPath(instant.getFileName(), content, false);
```

Comment:
@lamber-ken : thanks for the change. This should fix the empty content for clean requested but you still need the changes you did for commit and delta-commit requested instants. Those are still empty files and we nee the changes in HoodieCommitArchiveLog to handle it. Let me know if it makes sense ?

Reply:
Thanks, willing to fix empty commit and delta-commit files. I will ping when finished.

Reply:
hi @bvaradar, `HoodieCommitArchiveLog#convertToAvroRecord` don't need to handle commit and delta-commit empty files, because when reading these files, `HoodieCommitMetadata#fromBytes` checks whether empty content or not, return a new HoodieCommitMetadata instance when meets empty file.

BTW, from bellow structure, we can learn that creating a new instant with empty content.
![image](https://user-images.githubusercontent.com/20113411/71556992-8856fb80-2a7a-11ea-9c09-dc5e94743d2f.png)

Reply:
Makes sense for  not handling commit and deltacommit.  Will merge this PR.